### PR TITLE
test file modification

### PR DESCRIPTION
**Pull request by dengyufeng2206**
Signed-off-by: dengyufeng2206 <deng.yufeng@zte.com.cn>

#### What type of PR is this?
kind cleanup

#### What this PR does / why we need it:
ut modification
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/server/config_test.go [hunk 1]
```diff
@@ -139,7 +139,10 @@ func TestNewWithDelegate(t *testing.T) {
 	wrappingServer.PrepareRun()
 	wrappingServer.RunPostStartHooks(stopCh)
 
-	server := httptest.NewServer(wrappingServer.Handler)
+	server,err := httptest.NewServer(wrappingServer.Handler)
```
Comment (hzxuzhonghu) [inline staging/src/k8s.io/apiserver/pkg/server/config_test.go:142]: NewServer only returns one value.

Comment (dengyufeng2206) [reply staging/src/k8s.io/apiserver/pkg/server/config_test.go:142]: OK，thank you！

**Pull request closed without merging**